articles/lets-learn-about-osi-layer-model/index.md

```diff
@@ -0,0 +1,133 @@
+- Define which parts of the network their products should work with.
+- Communicate to users at which network layers their product operates 
+
+### Characteristics of OSI Model:
+
+OSI consists of several layers and each layer performs a particular network function. It's divided into two layers: upper layers and lower layers.
+
+The upper layer of the OSI model mainly deals with the application related issues, and they are implemented only in the software. The application layer is closest to the end-user. Both the end-user and the application layer interact with the software applications. An upper layer refers to the layer just above another layer.
+
+The lower layer of the OSI model deals with data transport issues. The data link layer and the physical layer are implemented in hardware and software. The physical layer is the lowest layer of the OSI model and is closest to the physical medium. The physical layer is mainly responsible for placing the information on the physical medium.
+
+OSI follows a vertical approach. ”If you can understand the OSI model and its layers, you can also then understand which protocols and devices can interoperate with each other when new technologies are developed and explained.
+
+- A layer should only be created where definite levels of abstraction are needed.
+- The function of each layer should be selected as per the internationally standardized protocols.
+- The number of layers should be large so that separate functions should not be put in the same layer. At the same time, it should be small enough so that architecture doesn't become very complicated.
+- In the OSI model, each layer relies on the next lower layer to perform primitive functions. Every level should able to provide services to the next higher layer.
+- Changes made in one layer should not need changes in other lavers.
+ 
+As mention, the model uses layers to help give a visual description of what is going on with a particular networking system. This can help network managers narrow. 
+
+### Functions of the OSI Layers
+There are seven OSI layers. Each layer has different functions. Each layer is assigned a particular task. OSI model divides the whole task into seven smaller and manageable tasks.  Each layer is self-contained so that tasks assigned to each layer can be performed independently. A list of seven layers are given below:
+- Physical Layer
+- Data-Link Layer
+- Network Layer
+- Transport Layer
+- Session Layer
+- Presentation Layer
+- Application Layer
+
+#### Physical layer
+- The main functionality of the physical layer is to transmit the individual bits from one node to another node.
+- It is the lowest layer of the OSI model, establishes, maintains, and deactivates the physical connection.
+- It specifies the mechanical, electrical, and procedural network interface specifications.
+
+##### Functions of a Physical layer:
+- Line Configuration: It defines the way how two or more devices can be connected physically.
```

Comment:
_Line Configuration_

Reply:
Do the same for all sub-titles mentioned in the article.

Reply:
Yeah I hope I resolve it